packages/builders/__tests__/components/actionRow.test.ts

```diff
@@ -0,0 +1,48 @@
+import { APIActionRowComponent, ButtonStyle, ComponentType } from 'discord-api-types/v9';
+import { ActionRow, ButtonComponent, createComponent } from '../../src';
+
+describe('Action Row Components', () => {
+	describe('Assertion Tests', () => {
+		test('GIVEN valid components THEN do not throw', () => {
+			expect(() => new ActionRow().addComponents(new ButtonComponent())).not.toThrowError();
```

Comment:
Let's add a test to ensure the output JSON from builders is valid

Reply:
Isn't that what the code for `GIVEN valid JSON input THEN valid JSON output is given` is doing?

Reply:
oh wait nvm I see